keeper/msg_server_portal.go

```diff
@@ -54,9 +55,14 @@ func (k portalMsgServer) Transfer(ctx context.Context, msg *portal.MsgTransfer)
 		return nil, portal.ErrPaused
 	}
 
-	peer, err := k.PortalPeers.Get(ctx, msg.Chain)
+	peer, err := k.PortalPeers.Get(ctx, msg.DestinationChainId)
 	if err != nil {
-		return nil, errors.Wrapf(portal.ErrInvalidPeer, "chain %d is not configured", msg.Chain)
+		return nil, errors.Wrapf(portal.ErrInvalidPeer, "chain %d is not configured", msg.DestinationChainId)
+	}
+
+	key := collections.Join(msg.DestinationChainId, msg.DestinationToken)
+	if has, _ := k.PortalSupportedBridgingPaths.Has(ctx, key); !has {
```

Comment:
do we want to ignore the error here?

Reply:
Similar to your above comment, the `Has` function will error if the key isn't found or if unmarshalling fails. However, it will always return false by default, which means it's safe to ignore the error as for both of those cases we should assume the element doesn't exist!